bin/build_sdb.py

```diff
@@ -5,18 +5,12 @@
 '''
 from __future__ import absolute_import, division, print_function
```

Comment:
(valid everywhere), do we still need those ?

Reply:
Not since we dropped Python 2 support. I'll do a clean up pass in a separate PR.